erpnext/accounts/doctype/journal_entry/journal_entry.py

```diff
@@ -22,8 +22,12 @@ def get_feed(self):
 		return self.voucher_type
 
 	def validate(self):
+		if self.voucher_type == 'Opening Entry':
+			self.is_opening = 'Yes'
+
 		if not self.is_opening:
 			self.is_opening='No'
```

Comment:
instead of two if conditions at line 25 and 28, use single line if condition
```
self.is_opening='Yes' if self.voucher_type == 'Opening Entry' else 'No'
```

Reply:
@rohitwaghchaure There are situations where user doesn't selects the voucher_type as 'Opening Entry' and keeps the voucher type as 'Journal Entry' in order to avoid fetching of defaults accounts and enters the accounts manually and just updates the `is_opening` as `Yes` manually. That is also a valid opening entry and in that case this condition will fail. So for that two conditions are needed.